packages/select/src/components/multi-select/multiSelect.tsx

```diff
@@ -406,6 +471,31 @@ export class MultiSelect<T> extends AbstractPureComponent<MultiSelectProps<T>, M
         };
     };
 
+    private getKeyDownHandler = (handleQueryListKeyDown: React.KeyboardEventHandler<HTMLElement>) => {
+        return (e: React.KeyboardEvent<HTMLElement>) => {
+            if (e.key === "Escape" || e.key === "Tab") {
+                if (this.popoverInputForCustomTarget != null) {
+                    this.popoverInputForCustomTarget.blur();
```

Comment:
is this still relevant for the version where the input is inside the popover? asking because it feels like this may just be copying the existing pattern